src/pages/uses/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Use Cases
+github: https://github.com/enactjs/docs/blob/develop/src/pages/uses/index.md
+---
+
+Enact has a wide variety of uses. Some of the more common use cases and the benefits Enact has for those applications are listed here. Don't see what you're looking for? Chances are if it's about using web technologies to produce native apps, the Enact can help. [Contact us](../contact/) to find out how.
+
+## Automotive
+
+![Image showing Enact Agate Themes](./agate-themes.jpg "A preview of automotive themes")
+
+Auto manufacturers, automotive platforms and companies working in this space are looking for new ways to leverage the ease and power of web application development. Enact has a proven track record of creating native quality application for web runtimes.
+
+[Read more about Enact for Automotive](./automotive/).
+
+## TV
+
+Enact has already shipped on millions of webOS Smart TVs across the world. With a full-featured component set, high performance and the capability to ship apps localized for any country, Enact is well positioned for developers looking to create apps for TV devices.
+
+## Robotics
+
+Robotics is another platform where creating engaging applications is becoming important. As robots interact with humans, a simple and intuitive approach to the user interface is required. Enact can help developers design applications that work well on a variety of interfaces and that communicate with the underlying systems.
```

Comment:
> ...work well on a variety of interfaces and that...

"Interface" is referenced in the preceding line. What if this were to say the following instead?

> ...work well on a variety of interaction models and that...